subscription-manager.spec

```diff
@@ -118,11 +118,8 @@ from the server. Populates /etc/docker/certs.d appropriately.
 %{_datadir}/rhsm/subscription_manager/plugin/container.py*
 # Copying Red Hat CA cert into each directory:
 %attr(755,root,root) %dir %{_sysconfdir}/docker/certs.d/cdn.redhat.com
-%attr(755,root,root) %dir %{_sysconfdir}/docker/certs.d/access.redhat.com
-%attr(755,root,root) %dir %{_sysconfdir}/docker/certs.d/registry.access.redhat.com
-%attr(644,root,root) %{_sysconfdir}/docker/certs.d/cdn.redhat.com/redhat-uep.crt
-%attr(644,root,root) %{_sysconfdir}/docker/certs.d/access.redhat.com/redhat-uep.crt
-%attr(644,root,root) %{_sysconfdir}/docker/certs.d/registry.access.redhat.com/redhat-uep.crt
+%attr(644,root,root) %{_sysconfdir}/rhsm/ca/redhat-entitlement-authority.pem
```

Comment:
Subscription-manager handles it just fine.  Yum is only getting a pointer to the redhat-uep.pem file so it shouldn't matter.  I can't vouch for ostree.